layouts/partials/head_includes.html

```diff
@@ -0,0 +1,85 @@
+<!-- CSS -->
+
+<style media="screen">
+  {{ partial "css/main.css" . | safeCSS }}
+</style>
+<style media="(min-width: 600px)">
+  {{ partial "css/min600px.css" . | safeCSS }}
+</style>
+<style media="(min-width: 769px)">
+  {{ partial "css/min769px.css" . | safeCSS }}
+</style>
+
+<noscript>
+  <link href="https://fonts.googleapis.com/css?family=Raleway:400,600,700" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Merriweather:300,300i,700,700i" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Ubuntu+Mono:400,700" rel="stylesheet">
+</noscript>
+
+{{ if .Site.Params.highlightjs }}
+  <style type="text/css" media="screen">
+    {{ partial "css/highlight.min.css" . | safeCSS }}
+  </style>
+{{ end }}
+
+{{ if .Site.Params.progressively }}
+  <style type="text/css" media="screen">
+    {{ partial "css/progressively.min.css" . | safeCSS }}
+  </style>
+{{ end }}
+
+{{ if .Site.Params.share }}
+  <style type="text/css" media="screen">
+    {{ partial "css/social-share-kit.css" . | safeCSS }}
+  </style>
+{{ end }}
+
+{{ if .Site.Params.colors }}
+  <style type="text/css" media="screen">
+    {{ partial "css/colors.css" . | safeCSS }}
+  </style>
+{{ end }}
+
+{{ range .Site.Params.extracssfiles }}
+  <link rel="stylesheet" href="{{ . | absURL }}">
+{{ end }}
+
+<!-- Icon -->
+<link rel="shortcut icon"
+{{ if .Site.Params.faviconfile }}
+    href="{{ .Site.Params.faviconfile | absURL }}"
+{{ else }}
+    href="{{ .Site.BaseURL }}img/favicon.ico"
+{{ end }}
+>
+
+<!-- RSS -->
+{{ if .RSSLink }}
+  <link href="{{ .RSSLink }}" rel="alternate" type="application/rss+xml" title="{{ .Site.Title }}" />
+{{ end }}
+
+<!-- Twitter metadata -->
```

Comment:
Wouldn't it make more sense for this section to live in the `meta` partial below?

Reply:
switched!